src/pages/__tests__/ProfileView.test.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { renderWithRouter } from 'test-utils';
+import {
+    BIOGRAPHY,
+    DISCORED_USERNAME,
+    EMAIL,
+    FIRST_NAME,
+    GITHUB_USERNAME,
+    LAST_NAME,
+    LINKS,
+    LOCATION,
+    PROFILE_IMAGE,
+    SOCIAL,
+    USERNAME
+} from '../../constants/user-constants';
+import ProfileView from '../ProfileView';
+import { useUser } from '../../contexts/UserContext';
+
+jest.mock('../../contexts/UserContext');
+const { UserProvider } = jest.requireActual('../../contexts/UserContext');
+
+const user = {
+    [EMAIL]: 'anEmail@adomain.com',
+    [SOCIAL]: {
+        [PROFILE_IMAGE]: 'src/image.png',
+        [USERNAME]: 'someuserName',
+        [FIRST_NAME]: 'Some',
+        [LAST_NAME]: 'LastName',
+        [LOCATION]: 'SomePlace, SomeLargerPlace',
+        [DISCORED_USERNAME]: 'DicordingName',
+        [GITHUB_USERNAME]: 'GithubberName',
+        [BIOGRAPHY]: 'Lines of the bios',
+        [LINKS]: ['url/url', 'url2/url2']
+    }
+}
+
+describe('verify correct values are displayed from user context', () => {
```

Comment:
Depending on what's actually required, you might also want to test for whether the view renders without crashing when some of the user information is not present.

Reply:
yea I was debating adding that, I just wasn't sure how i wanted to verify it. I guess I could make the defaults i setup as constants and check that they're there in the test.